google-map.html

```diff
@@ -331,6 +331,14 @@
       additionalMapOptions: {
         type: Object,
         value: function() { return {}; }
+      },
+
+      /**
+       * The markers on the map.
+       */
+      markers: {
+        type: Array,
+        value: function() { return []; }
```

Comment:
we should make this `readOnly: true`. That means you'll need to use the magic `this._setMarkers(val)`